dev/test/specs/starter-test.js

```diff
@@ -78,12 +78,25 @@ function loadSeleniumUtils(cdnBranch) {
     script.src = `${cdnBranch}/test/selenium-utils.js`;
     document.getElementsByTagName('head')[0].appendChild(script);
   }, cdnBranch);
+  browser.waitUntil(() => {
+    try {
+      const result = browser.execute('pierceShadows(["head"])');
```

Comment:
Should we check result for an error or whatever expected value might be? Also, why do we pierce looking for `head` element, is that just arbitrary and we use success as a sign that `pierceShadows()` has loaded? If so also seems worth a comment.